src/style.css

```diff
@@ -0,0 +1,84 @@
+html {
+  font-family: Arial, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  height: 300px;
+  line-height: 300px;
+  text-align: center;
+}
+
+.block--1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .blocks {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-basis: calc(100% / 2);
```

Comment:
if you have many calc functions it might get heavy for the browser. here you can simply use 50%